asciidoc-confluence-publisher-converter/src/main/java/org/sahli/asciidoc/confluence/publisher/converter/AsciidocConfluencePage.java

```diff
@@ -41,6 +41,7 @@
 import static java.util.Collections.emptyMap;
 import static java.util.Collections.unmodifiableMap;
 import static org.asciidoctor.Asciidoctor.Factory.create;
+import static org.asciidoctor.SafeMode.SAFE;
```

Comment:
The default previously used was `SECURE` which is even more restrictive than `SAFE` and does not allow including other asciidoc files at all (but uses links instead). If needed later, this property could be relaxed even more (which might indeed be needed for supporting includes of images and asciidoc files outside of the parent / base dir).